src/consts/env_defaults.py

```diff
@@ -11,7 +11,7 @@
 DEFAULT_SSH_PUBLIC_KEY_PATH: Path = Path.home() / ".ssh" / "id_rsa.pub"
 DEFAULT_INSTALLER_KUBECONFIG = None
 DEFAULT_LOG_FOLDER: Path = Path("/tmp/assisted_test_infra_logs")
-DEFAULT_IMAGE_TYPE: str = consts.ImageType.FULL_ISO
+DEFAULT_IMAGE_TYPE: str = consts.ImageType.MINIMAL_ISO
```

Comment:
As a result of this change, all of our test-infra jobs as well as QE e2e tests will be affected.
Need also QE approval - FYI @lalon4 @eliajahshan

Reply:
Thanks for catching that and pinging QE!

Reply:
Rebumping this @lalon4 @eliajahshan when either of you get a chance, may I have your review

Reply:
@eliorerz QE has approved!